app/presenters/piv_cac_error_presenter.rb

```diff
@@ -0,0 +1,72 @@
+class PivCacErrorPresenter
```

Comment:
combines the `PivCacAuthenticationSetupErrorPresenter` and `PivCacAuthenticationLoginPresenter`